pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sphinx_gherkindoc"
-version = "3.3.1"
+version = "3.3.3"
```

Comment:
Made this .3 in anticipation of #21 getting looked at/merged first. Maybe that's a bad idea, but I dislike merge conflicts _that_ much 😄

Reply:
I think 21 should merge first regardless :-)

Reply:
that merged, so all good